randovania/__main__.py

```diff
@@ -17,6 +17,12 @@ def main() -> None:
 
     logging.debug("Starting Randovania...")
 
+    if randovania.VERSION == randovania.UNKNOWN_VERSION or randovania.GIT_HASH == randovania.UNKNOWN_GIT_HASH:
+        logging.info(
```

Comment:
I'd say warning, since it does indicate that something is wrong with the environment even if we're trying to accomodate the error